ipaclient/plugins/vault.py

```diff
@@ -752,8 +751,7 @@ def forward(self, *args, **options):
                 error=_('Invalid vault type'))
 
         # initialize NSS database
-        current_dbdir = paths.IPA_NSSDB_DIR
```

Comment:
I stand corrected. But it looks suspicious anyway. I think that the original intent was to actually use `nsslib.current_dbdir` and that the local variable is used by mistake. But given that nobody complained about NSS initialization failures in vault so far, it is probably OK this way.